common/djangoapps/student/forms.py

```diff
@@ -75,7 +81,8 @@ def save(
                 'token': token_generator.make_token(user),
                 'protocol': 'https' if use_https else 'http',
                 'platform_name': configuration_helpers.get_value('platform_name', settings.PLATFORM_NAME),
-                'contact_email': settings.CONTACT_EMAIL
+                'contact_email': settings.CONTACT_EMAIL,
```

Comment:
@lukyfebrianto what's the difference between `CONTACT_EMAIL ` and `LABSTER_TECH_SUPPORT_LINK `?
Why don't you use one of them everywhere?

Reply:
@TamoshaytisV `LABSTER_TECH_SUPPORT_LINK` is a link not an email address.
see, https://github.com/Livit/Livit.Learn.EdX/blob/labster/develop/lms/envs/labster.py#L32